yaksh/forms.py

```diff
@@ -200,7 +200,7 @@ class CourseForm(forms.ModelForm):
     class Meta:
         model = Course
         fields = ['name', 'active', 'enrollment']
-
+        exclude = ["is_trial"]
```

Comment:
Suggestion - Not exactly needed since fields already will only show the above mentioned fields